tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -11,145 +11,376 @@
+                                                                                        11, 12, 13, 14,
+                                                                                        15, 16, 98, 99)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='CITIZENSHIP_STATUS', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='COOPERATION_CHILD_SUPPORT', result_function=validators.oneOf((1, 2, 9)),
+                ),
+            # TODO THIS ISNT EXACTLY RIGHT SINCE FED TIME LIMIT IS A STRING.
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                      result_field='MONTHS_FED_TIME_LIMIT', result_function=validators.isLargerThan(1),
+                ),
+            # TODO THIS ISNT EXACTLY RIGHT SINCE FED TIME LIMIT IS A STRING.
+            validators.if_then_validator(
+                      condition_field='RELATIONSHIP_HOH', condition_function=validators.oneOf((1, 2)),
+                      result_field='MONTHS_FED_TIME_LIMIT', result_function=validators.isLargerThan(1),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='EMPLOYMENT_STATUS', result_function=validators.oneOf((1, 2, 3)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                      result_field='WORK_PART_STATUS', result_function=validators.oneOf((1, 2, 3)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                      result_field='WORK_ELIGIBLE_INDICATOR', result_function=validators.oneOf((1, 2, 3, 4,
+                                                                                                5, 6, 7, 8,
+                                                                                                9, 12)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                      result_field='WORK_PART_STATUS', result_function=validators.oneOf((1, 2, 5, 7, 9,
+                                                                                         15, 16, 17, 18, 19,
+                                                                                         99)),
```

Comment:
`WORK_ELIGIBLE_INDICATOR` and `WORK_PART_STATUS` are potentially simplify-able as well